### ISSUE TITLE
Include an image of ESA ExoMy

### ISSUE THREAD

**Issue by sethfischer** [sethfischer/osr#3]
Include an image of _ESA ExoMy_ in the _Related projects_ section.

# Suggested image

https://www.esa.int/ESA_Multimedia/Images/2020/11/ExoMy

![esa-exomy](https://user-images.githubusercontent.com/2492233/158971175-e4e8bbd9-c6a2-4130-9464-b1986ddd607e.jpeg)

**Comment by sethfischer** [comment]
@esa-prl May I have your permission to include this image in the _Related projects_ section? This section lists a number of Mars rover projects, including _ESA ExoMy_.

I would add EXIF metadata tags to JPEG image similar to the following:

```
Artist           : [please advise]
Attribution Name : [please advise]
Attribution URL  : https://www.esa.int/ESA_Multimedia/Images/2020/11/ExoMy
License          : [please advise]
Description      : Six-wheeled ExoMy rover designed by ESA's Planetary Robotics Laboratory
Rights           : European Space Agency Planetary Robotics Laboratory
Usage Terms      : [please advise]
Comment          : ESA ExoMy rover
```

**Comment by sethfischer** [comment]
Authorisation granted on 2022-04-25 by European Space Agency (ESA) via private correspondence.

Image copyright: European Space Agency (ESA).
Image source: https://www.esa.int/ESA_Multimedia/Images/2020/11/ExoMy
ESA reference: ESA HQ PHOTOS 20220425-00239

Committed with the following EXIF metadata tags:

```
Artist           : European Space Agency (ESA)
Attribution Name : European Space Agency (ESA)
Attribution URL  : https://www.esa.int/ESA_Multimedia/Images/2020/11/ExoMy
Description      : Six-wheeled ExoMy rover designed by ESA's Planetary Robotics Laboratory.
Usage Terms      : Copyright European Space Agency (ESA)
Comment          : ESA ExoMy rover
```



Closed by 5d187efad37befddb033534ae7b534f50e7c5ea2
